Sprint-1/exercises/paths.js

```diff
@@ -15,4 +15,8 @@ const base = filePath.slice(lastSlashIndex + 1);
 console.log(`The base part of ${filePath} is ${base}`);
 
 // Create a variable to store the dir part of the filePath variable
+const dir = filePath.slice(0, lastSlashIndex);
+
 // Create a variable to store the ext part of the variable
+const extPart = base.slice(base.lastIndexOf('.')+1)
```

Comment:
Look at the diagram above and see if it matches up exactly with your solution.

Reply:
Yes, after reviewing the diagram and running the code it matches,  please let me know if solution not aligns correctly

Reply:
Please take note of all the _characters_ under the 'base' column